### DISCUSSION TITLE
Datatable row_group not working as expected

### DISCUSSION THREAD

**Discussion by marcorovida** [primefaces/community#336] [PrimeReact] [answered]
Hi everyone, 
I'am using datatabel with row group options (https://primereact.org/datatable/#row_group) that require to work properly as shown in the component page: sortField + groupRowsBy props with the same valuesame field data.

For some reason it seems that it work well of first rendering, but if i try to change values (with my custom select) sono groups are duplicated.
I tryied to set multisortmeta and sortMode as multiple too but without difference.

First load page  grouped by Name
![FireShot Capture 001 - Create Next App - localhost](https://github.com/primefaces/community/assets/8331253/52595adf-dd8b-4d0b-8993-b29673191b20)

After selecting Date as group mode
![FireShot Capture 002 - Create Next App - localhost](https://github.com/primefaces/community/assets/8331253/82d364a6-1d52-43bb-9317-5a4ffe8709de)

As show in yellow, some rows/group are splitted. Same result using Timestamp too.
I am doing something wrong or it's a bug?

Thanks for any suggestions

**Comment by marcorovida** [comment, marked as answer]
UPDATE : For some reason i can't explain, calling the datatable  reset() method,  fix the problem. Nothin happen to my custom ordering values but the row groups are shown as expected after reset() execution. Is not a good solution because i bet that appliyng some filters (not in my  example above), these will be reset with initial state.

**Reply by melloware** [reply]
If you update this StackBlitz to show your issue I can take a look: https://stackblitz.com/edit/react-fdjgqs?file=src%2FApp.js
